include/cuco/static_map.cuh

```diff
@@ -649,12 +649,14 @@ class static_map {
      *
      * @tparam CG The type of the cooperative thread group
      * @param g The ooperative thread group used to copy the slots
+     * @param source_device_view `device_view` to copy from
      * @param memory_to_use Array large enough to support `capacity` elements. Object does not take the ownership of the memory
-     * @return Copy of static map
+     * @return Copy of passed `device_view`
      */
     template <typename CG>
-    __device__ device_view make_copy(CG g,
-                                     pair_atomic_type* const memory_to_use) const noexcept
+    __device__ static device_view make_copy(CG g,
+                                            device_view const& source_device_view,
```

Comment:
Since `device_view` is a view type, it's usually passed by value. 
```suggestion
                                            device_view source_device_view,
```

Reply:
done